src/Command/BakeFixtureFactoryCommand.php

```diff
@@ -420,6 +420,8 @@ public function getOptionParser(): ConsoleOptionParser
         $name = ($this->plugin ? $this->plugin . '.' : '') . $this->name;
         $parser = new ConsoleOptionParser($name);
 
+        $parser = $this->_setCommonOptions($parser);
```

Comment:
This fixed the error since the common option's default values aren't set without this call.